tests/libtest.sh

```diff
@@ -516,12 +544,15 @@ os_repository_new_commit ()
     cd ${test_tmpdir}
 }
 
-# Usage: if ! skip_one_without_user_xattrs; then ... more tests ...; fi
-_have_user_xattrs=''
-have_user_xattrs() {
+assert_has_setfattr() {
     if ! which setfattr 2>/dev/null; then
         fatal "no setfattr available to determine xattr support"
     fi
+}
+
+_have_user_xattrs=''
+have_user_xattrs() {
+    assert_has_setfattr
```

Comment:
Minor, but maybe move the assertion below the test so we don't call it every time?